src/assets/handhelds/paper_note/paper_note.asset.ts

```diff
@@ -0,0 +1,101 @@
+DefineAsset({
+	name: 'Paper Note',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		paper: {
+			name: 'Paper',
+			default: '#F2E5AE',
+		},
+	},
+	// size:200, y:304, X:153
+	preview: 'preview.png',
+	attributes: {
+		provides: [
+			'Handheld',
+		],
+		requires: [
+			'!Hand_restricting_cover',
+		],
+	},
+	modules: {
+		location: {
+			type: 'typed',
+			name: 'Location of the note',
+			variants: [
+				{
+					id: 'somewhere',
+					name: 'Kept somewhere (worn without graphics)',
+					default: true,
+
+				},
+				{
+					id: 'right',
+					name: 'Held in the right hand',
+					properties: {
+						poseLimits: {
+							rightArm: {
+								fingers: 'fist',
+							},
+							options: [
+								{
+									rightArm: {
+										rotation: 'up',
+									},
+								},
+								{
+									rightArm: {
+										rotation: 'down',
+									},
+								},
+							],
+						},
```

Comment:
This can be simplified a lot (similarly the one below):
```suggestion
						poseLimits: {
							rightArm: {
								fingers: 'fist',
								rotation: ['up', 'down'],
							},
						},
```

Reply:
All instances on all handhelds improved accordingly.